autovalet_description/readme.MD

```diff
@@ -1,3 +1,5 @@
+# autovalet_description
+
 This is the autovalet husky URDF description package. We have a modified version of the husky with a LiDAR and 2 d435s on it. The huskyValet_urdf launch file calls xacro parser onto the modified urdf passed to it. Also, it runs robot_state publisher and joint_state publisher for standalone vizualization purposes.
 
```

Comment:
I think the readme here should go sth like this:
This is the autovalet husky URDF package. We have a modified version of the husky with a LiDAR and 2 Realsense D435s on it. We call the standard husky URDF with out autovalet.urdf.xacro as an extra. Our sensor transforms are defined in the xacro file. The URDF is published to ROS by launching the autovalet_description.launch. In order to view the robot model generated by the URDF, please launch robot state publisher and the joint state publisher along with the autovalet_description.launch file.